frontend/test/Copy.test.tsx

```diff
@@ -0,0 +1,46 @@
+import { afterEach, beforeAll, expect, it, vi } from 'vitest'
+import { cleanup, fireEvent, render, waitFor } from '@testing-library/react'
+import Search from '@pages/search'
+
+import CopyFixture from './fixtures/Copy.json'
+
+beforeAll(() => {
+  vi.mock('next/config', () => ({
+    default: () => ({
+      publicRuntimeConfig: {
+        KITSPACE_GITEA_URL: 'https://gitea.kitspace.test',
+        KITSPACE_ASSET_URL: 'https://s3.kitspace.test',
```

Comment:
We don't currently use a `s3.kitspace.test` domain.

Reply:
Resolved in 2c0fcbb9.